superset-frontend/src/explore/components/controls/withAsyncVerification.tsx

```diff
@@ -142,12 +142,14 @@ export default function withAsyncVerification({
     } = props;
     const otherPropsRef = useRef(restProps);
     const [verifiedProps, setVerifiedProps] = useState({});
+    const verifiedPropsRef = useRef(verifiedProps);
```

Comment:
Can't we remove `verifiedProps` state and just use `verifiedPropsRef`?

Reply:
`verifiedProps` is necessary for rendering a new options list through `setVerifiedProps`.
I have an idea that this part can be handled using `useEffectEvent` without utilizing additional ref access.